builtin/common/after.lua

```diff
@@ -16,18 +19,23 @@ core.register_globalstep(function(dtime)
 		if time >= job.expire then
 			core.set_last_run_mod(job.mod_origin)
 			job.func(unpack(job.arg))
-			table.remove(jobs, i)
+			jobs[i] = jobs[#jobs]
+			jobs[#jobs] = nil
```

Comment:
Use a variable for `#jobs` to avoid calculating it twice.

Reply:
Done.